docs/en/token_endpoint.rst

```diff
@@ -295,3 +295,39 @@ The claims available in the *ID Token* are given below.
  - https://openid.net/specs/openid-igov-openid-connect-1_0-03.html#rfc.section.3.1
 
 
+.. _TOKEN_ENDPOINT_ERRORS:
+
+Errors
+------
+
+.. list-table:: 
+   :widths: 20 60 20
+   :header-rows: 1
+
+   * - **Claim**
+     - **Description**
+     - **Supported by**
+
+   * - *invalid_client*
```

Comment:
In OIDC Core questo codice fa riferimento alla Token Error Response
Ref:
RFC 6749 - Oauth 2.0 – 5.2
OIDC Core 1.0 – 3.1.3.4